containers/ruby/.devcontainer/library-scripts/ruby-debian.sh

```diff
@@ -247,25 +247,48 @@ git clone --depth=1 \
     -c fetch.fsck.zeroPaddedFilemode=ignore \
     -c receive.fsck.zeroPaddedFilemode=ignore \
     https://github.com/rbenv/rbenv.git /usr/local/share/rbenv
-ln -s /usr/local/share/rbenv/bin/rbenv /usr/local/bin
-updaterc 'eval "$(rbenv init -)"'
+
 git clone --depth=1 \
     -c core.eol=lf \
     -c core.autocrlf=false \
     -c fsck.zeroPaddedFilemode=ignore \
     -c fetch.fsck.zeroPaddedFilemode=ignore \
     -c receive.fsck.zeroPaddedFilemode=ignore \
     https://github.com/rbenv/ruby-build.git /usr/local/share/ruby-build
+
 mkdir -p /root/.rbenv/plugins
-ln -s /usr/local/share/ruby-build /root/.rbenv/plugins/ruby-build
+
 if [ "${USERNAME}" != "root" ]; then
     mkdir -p /home/${USERNAME}/.rbenv/plugins
-    chown -R ${USERNAME} /home/${USERNAME}/.rbenv
-    ln -s /usr/local/share/ruby-build /home/${USERNAME}/.rbenv/plugins/ruby-build
 fi
 
+# Update bashrc and zshrc with conditionals
+# We use "not false" so that the default state is true, i.e., enabled
+updaterc 'if [ "${RVM_ENABLED}" != "false" ]; then 
+    source /usr/local/rvm/scripts/rvm 
+fi'
+
+# All commands have the output redirected to /dev/null so that there is no output on shell startup.
+if [ "${USERNAME}" != "root" ]; then
+    mkdir -p /home/${USERNAME}/.rbenv/plugins
+    updaterc 'if [ "${RBENV_ENABLED}" != "false" ]; then
+        chown -R ${USERNAME} /home/${USERNAME}/.rbenv
+        ln -s /usr/local/share/ruby-build /home/${USERNAME}/.rbenv/plugins/ruby-build > /dev/null 2>&1
+    else
+        rm -f /home/${USERNAME}/.rbenv/plugins/ruby-build > /dev/null 2>&1
+    fi'
+fi
+
+updaterc 'if [ "${RBENV_ENABLED}" != "false" ]; then
+        sudo ln -s /usr/local/share/rbenv/bin/rbenv /usr/local/bin > /dev/null 2>&1
+        eval "$(rbenv init -)"
+        sudo ln -s /usr/local/share/ruby-build /root/.rbenv/plugins/ruby-build > /dev/null 2>&1
+    else
+        rm -f  /root/.rbenv/plugins/ruby-build > /dev/null 2>&1
+        rm -f  /usr/local/bin/rbenv > /dev/null 2>&1
```

Comment:
See previous comment on removing the symlinks. I would only remove things for the current user. Permissions can get messed up here.  

Honestly, I also would just leave the .rbenv folder.  It does no harm to be there.